src/amo/components/AddonMoreInfo/index.js

```diff
@@ -269,10 +269,15 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
   }
 }
 
-export const mapStateToProps = (state: AppState) => {
+export const mapStateToProps = (state: AppState, ownProps: Props) => {
+  const { addon } = ownProps;
   return {
-    userId: state.users.currentUserID,
     hasStatsPermission: hasPermission(state, STATS_VIEW),
+    userId: state.users.currentUserID,
+    version:
```

Comment:
nit: could we still call this `currentVersion`? that name would still be relevant right? I also think `version.version` (above) would be a bit clearer if the object and property name were different..

What do you think?

Reply:
Good suggestion, I like it! I've made that change too.